scripts/convert_csv_to_fasta.py

```diff
@@ -0,0 +1,84 @@
+# Script to convert csv files to fasta files
+#
+import csv
+
+def separate_seq(sequence:str) -> str:
+    """ Separate sequence in 60-characters chunks
+         and append new line to each chunk except remaing one
+
+    Args:
+        sequence : protein sequence from csv file
+
+    Returns:
+        fs: separated sequence for fasta file
+    """
+    # Initialize new sequence
+    fs = ""
+    # Count number of full chunks
+    chunk = 60
+    d = len(sequence) // chunk
+    r = len(sequence) % chunk
+    new_line = '\n'
+    # Iterate hrough chunks and append new line char - \n
+    for i in range(d):
+        low = i * chunk
+        high = (i + 1) * chunk
+    # If the last line is full chunk, no new line char
+        if r == 0 and i == d - 1:
+            new_line = ""
+        fs += f"{sequence[low : high]}{new_line}"
+    # Append remainder
+    fs += f"{sequence[d * chunk :]}"
+    return fs
+
+
+def csv_to_fasta(csv_file, fasta_file):
+    """ Convert csv fils to fasta files
+
+    Args:
+        csv_file: csv formatted file
```

Comment:
You should probably also add fasta_file to the list of Args. Maybe also specify that it's just a filename to write to, since no such file needs to exist before running this function.